draft-wang-ppm-differential-privacy.md

```diff
@@ -229,8 +229,14 @@ privacy guarantee. If the DP guarantee is achieved with a minimum batch size
 number of Clients adding local DP noise, and minimum batch size is not reached
 when a data collection task expires, each Aggregator can add the remaining noise
 by generating the same local DP noise, on the missing number of Clients being
-the gap between actual number of Clients and minimum batch size.
+the gap between actual number of Clients and minimum batch size. For example:
 
+* {{FMT20}} describes a way to quantify the privacy guarantee in terms of
+  approximate-DP guarantee. The overall privacy guarantee is amplified by
+  aggregation of Clients' measurements with local DP noise with ϵ-DP guarantee.
+
+* {{FMT22}} further tightens the privacy amplification analysis in terms of
+  Rényi-DP guarantee.
```

Comment:
Pointers to analysis is useful in security considerations, but at this point all we really care about is intuition for what we get. Is the primary motivation is to be able to aggregate a batch that is not yet "full"? Are there any other benefits?